opus/application/static_media/js/hash.js

```diff
@@ -47,14 +47,19 @@ var o_hash = {
                         // opus.prefs.view = search, browse, cart, or detail
                         // opus.prefs.browse =  'gallery' or 'dataTable',
                         let page = o_browse.getCurrentPage();
-                        hash.push("page=" + page);
+                        // we will no longer have page in url, so no need to push page
```

Comment:
As discussed, we need to leave page in there for now. Once Debby integrates this onto her startobs branch, this case should be removed entirely because page is no longer a valid slug.